pkg/apis/tenancy/v1alpha1/helper/helper_test.go

```diff
@@ -143,18 +143,6 @@ func TestWorkspaceKey(t *testing.T) {
 		ws   string
 		want string
 	}{
-		{
```

Comment:
Testify has some assertions for panics, doesn't it?

Reply:
Done.